packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -449,20 +449,14 @@ export const schema: SchemaType<DbTag> = {
   'canVoteOnRels.$': {
     type: String,
   },
-
-  subforumShortformPostId: {
```

Comment:
This caused a bunch of errors in LW production, because of old clients continuing to query this field until they refresh.